p2p/src/main/kotlin/maru/p2p/MaruPeerManager.kt

```diff
@@ -67,19 +63,16 @@ class MaruPeerManager(
     if (scheduler == null) {
       scheduler = Executors.newSingleThreadScheduledExecutor(Thread.ofPlatform().daemon().factory())
     }
-    scheduler!!.scheduleAtFixedRate(
-      /* command = */ this::logConnectedPeers,
-      /* initialDelay = */ 5,
-      /* period = */ 20,
-      /* unit = */ TimeUnit.SECONDS,
-    )
+    scheduler!!.scheduleAtFixedRate({
```

Comment:
```suggestion
  scheduler!!.scheduleAtFixedRate(
      /* command = */ this::logConnectedPeers,
      /* initialDelay = */ 5,
      /* period = */ 20,
      /* unit = */ TimeUnit.SECONDS,
    )
```

Imo is more readable and 20s delay is feels too long on start

Reply:
Yes, I fixed it already